### ISSUE TITLE
Update getting started at EUR

### ISSUE THREAD

**Issue by DorienHuijser** [eur-synclab/eur-synclab.github.io#60]
Many EUR portals are not accessible anymore from home, you now need MyApps to access them: I should add that + a link to a myapps manual

**Comment by eduardklap** [comment]
Good point. Might add that using EduVPN when surfing is also an option for accessing many of the EUR portals.

**Comment by DorienHuijser** [comment]
Oeh I didn't know that!
Another thing for the list: Metis will become Pure, would be nice to update that immediately as well
